ui/src/app/utils/access-utils.tsx

```diff
@@ -60,8 +60,8 @@ export const redirectToNiH = (): void => {
   AnalyticsTracker.Registration.ERACommons();
   const url = serverConfigStore.get().config.shibbolethUiBaseUrl + '/login?return-url=' +
       encodeURIComponent(
-        window.location.origin.toString() + NIH_CALLBACK_PATH + '?token=<token>');
-  window.open(url, '_blank');
+        window.location.origin.toString() + NIH_CALLBACK_PATH + '?token={token}');
+  window.open(serverConfigStore.get().config.rasLogoutUrl + url, '_blank');
```

Comment:
nit: prefer to construct the `url` including `rasLogoutUrl` so this line can be unchanged